src/target/efm32.c

```diff
@@ -349,79 +344,59 @@ typedef struct efm32_v2_di_miscchip_t {
 /* pkgtype */
 typedef struct efm32_v2_di_pkgtype_t {
 	uint8_t pkgtype;
-	char* name;
+	char *name;
 } efm32_v2_di_pkgtype_t;
 
 efm32_v2_di_pkgtype_t const efm32_v2_di_pkgtypes[] = {
-	{74, "WLCSP"},	/* WLCSP package */
-	{76, "BGA"},	/* BGA package */
-	{77, "QFN"},	/* QFN package */
-	{81, "QFxP"},	/* QFP package */
+	{74, "WLCSP"}, /* WLCSP package */
+	{76, "BGA"},   /* BGA package */
+	{77, "QFN"},   /* QFN package */
+	{81, "QFxP"},  /* QFP package */
 };
 
 /* tempgrade */
 typedef struct efm32_v2_di_tempgrade_t {
 	uint8_t tempgrade;
-	char* name;
+	char *name;
 } efm32_v2_di_tempgrade_t;
 
 efm32_v2_di_tempgrade_t const efm32_v2_di_tempgrades[] = {
 	{0, "-40 to 85degC"},
 	{1, "-40 to 125degC"},
 	{2, "-40 to 105degC"},
-	{3, "0 to 70degC"}
+	{3, "0 to 70degC"},
 };
 
 /* -------------------------------------------------------------------------- */
 /* Helper functions - Version 1 V1 */
 /* -------------------------------------------------------------------------- */
 
-/**
- * Reads the EFM32 Extended Unique Identifier EUI64 (V1)
- */
+/* Reads the EFM32 Extended Unique Identifier EUI64 (V1) */
 static uint64_t efm32_v1_read_eui64(target *t)
 {
 	uint64_t eui;
 
-	eui  = (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_1) << 32;
-	eui |= (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_0) <<  0;
+	eui = (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_1) << 32;
+	eui |= (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_0) << 0;
 
 	return eui;
 }
 
-#if 0
-/**
- * Reads the EFM32 Extended Unique Identifier EUI48 (V2)
- */
-static uint64_t efm32_v2_read_eui48(target *t)
-{
-	uint64_t eui;
-
-	eui  = (uint64_t)target_mem_read32(t, EFM32_V2_DI_EUI48H) << 32;
-	eui |= (uint64_t)target_mem_read32(t, EFM32_V2_DI_EUI48L) <<  0;
-
-	return eui;
-}
-#endif
-/**
- * Reads the Unique Number (DI V2 only)
- */
+/* Reads the Unique Number (DI V2 only) */
 static uint64_t efm32_v2_read_unique(target *t, uint8_t di_version)
 {
 	uint64_t unique;
 	switch (di_version) {
 	case 2:
-		unique  = (uint64_t)target_mem_read32(t, EFM32_V2_DI_UNIQUEH) << 32;
+		unique = (uint64_t)target_mem_read32(t, EFM32_V2_DI_UNIQUEH) << 32;
```

Comment:
Given how this code is shaped and written, it makes sense to move the definition of `unique` into this block and drop the `<< 0` on the next line.

Additionally, the cast for the second line is not required - it's only required on this definition line.